### ISSUE TITLE
Problem with bluetooth

### ISSUE THREAD

**Issue by ShashankArora92** [pop-os/pop#1626]
My Pop os Bluetooth button is not working.
I have tried "rfkill" to unblock Bluetooth. Everything is unblocked when checked using "rfkill list".
Every time I have to use Bluetooth I have to use the command "sudo systemctl restart bluetooth".
Please help me with this problem.
Thank You

**Comment by bflanagin** [comment]
@ShashankArora92: We'll need more info, reply with the output from `cat /etc/os-release` and if possible give us the make and model of you computer. 

Also what Bluetooth button are you referring too? Could you supply screen shots or a more detailed description of what you are attempting to make work before using "sudo systemctl restart bluetooth"?

**Comment by leviport** [comment]
Isn't this a duplicate of https://github.com/pop-os/pop/issues/1623?

**Comment by ShashankArora92** [comment]
Yes...we both are facing the same issue.

**Comment by leviport** [comment]
Let's not open multiple issues about the same bug then. We can work on a solution in the the other issue.
